app/views/membership.html

```diff
@@ -0,0 +1,292 @@
+<project-header class="top-header"></project-header>
+<project-page class="membership" ng-if="project">
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header">
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+        </div>
+      </div>
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-12">
+              <h1>
+                Membership
+                <small class="pull-right meta">
+                  <a href="{{'roles' | helpLink}}">
+                    Learn more <i class="fa fa-external-link"></i>
+                  </a>
+                </small>
+              </h1>
+              <alerts alerts="alerts"></alerts>
+            </div>
+          </div>
+          <div  ng-if="!('rolebindings' | canI : 'list')">
+            <p>Nothing to see here, move along folks.</p>
+          </div>
+          <uib-tabset
+            ng-if="'rolebindings' | canI : 'list'">
+            <uib-tab
+              ng-repeat="(subjectKindName, subjectKind) in subjectKindsForUI | orderBy: 'sortOrder'"
```

Comment:
I don't see you using subjectKindName anywhere, you should just be able to do `subjectKind in subjectKindsForUI`, that does what you want for maps where you dont care about the key